.rubocop.yml

```diff
@@ -64,7 +64,7 @@ Lint/AmbiguousOperator:
 # TODO: Change the checking so that if the variable being assigned to has
 # a value ALREADY, then raise an error.
 Lint/AssignmentInCondition:
-  Enabled: false
+  Enabled: true
```

Comment:
This cop is enabled by default, so we shouldn't have to explicitely enable it here. However, if you do want to continue to explicitely enable it here, we should update the comment
https://rubocop.readthedocs.io/en/latest/cops_lint/#lintassignmentincondition
https://github.com/rubocop-hq/ruby-style-guide#safe-assignment-in-condition

If we decide to keep it explicitely enabled, should we allow safe assignment?
```
if (v = array.grep(/foo/))
  do_something(v)
  # some code
end
```
^^ would be allowed, as safe assignment defaults to true.

Reply:
I'm 👍 to removing enabled cops, and only listing disabled ones